### PR TITLE
chore(agd): upgrade all orchestration vats to new liveslots

### LINKED ISSUE #10762: Restart Vats in u18 to Address watchPromise Leak
## What is the Problem Being Solved?
There is a concern that certain vats in u18 (ibc, localchain, network, and orchestration) may exhibit memory leaks due to the watchPromise issue. Restarting these vats with the applied fixes would prevent leaks when they are utilized.

## Description of the Design
The solution involves restarting the identified vats (ibc, localchain, network, and orchestration) to ensure they pick up the fixes for the watchPromise leak. For the orchestration vat, there is a core proposal already prepared to facilitate the upgrade. Other vats might require similar proposals or equivalent upgrade mechanisms to achieve the desired outcome.

> The proposal exists for Orch Core (and was formerly run in an earlier upgrade.go) like this:
```
+                               vm.CoreProposalStepForModules(
+                                       // Upgrade to new liveslots for repaired vow usage.
+                                       "@agoric/builders/scripts/vats/upgrade-orch-core.js",
+                               ),
```
## Security Considerations
- Validate that the core proposals are correctly configured to avoid errors during the upgrade.
- Verify the idempotency of the upgrade scripts to prevent unexpected behavior in case of re-execution.

## Scaling Considerations

- Confirm that restarting these vats does not disrupt ongoing operations or degrade the performance of other components in the system.
- Assess the impact of the restart on dependent modules or transactions to avoid potential bottlenecks or delays.

## Test Plan
- Validate the fixes for the watchPromise leak in a staging environment before deploying to production.
- Test the core proposal scripts (@agoric/builders/scripts/vats/upgrade-orch-core.js and related scripts) to confirm their idempotency and correct execution.
- Monitor the system after the restart to ensure stability and no memory leaks.

## Upgrade Considerations

- Ensure that all core proposals are ready and tested prior to execution.
- Coordinate the restart to minimize disruption, potentially scheduling it during low-traffic periods.
- Maintain a rollback plan in case the upgrade introduces unforeseen issues.
- Document the upgrade process and any dependencies to streamline future updates.

### PR DESCRIPTION
**Pull request by michaelfig**
refs: #10758, #10756
closes: #10762

## Description

Upgrade the Orchestration (both Core and API) vats to use liveslots with the #10758 fix for a `watchPromise` memory leak (#10756).

### Security Considerations

No code change to the vat user code in question, just to the liveslots it relies on.

### Scaling Considerations

Should flatten the memory growth curve.

### Documentation Considerations

No changes to existing data.

### Testing Considerations

The upgraded vats should be monitored for memory growth.

### Upgrade Considerations

n/a

### CODE REVIEW

File: packages/builders/scripts/vats/upgrade-orchestration.js
Comment (mhofman) [inline packages/builders/scripts/vats/upgrade-orchestration.js:1]: It looks like `upgrade-orch-core-proposal.js` really is a generic `upgrade-named-vats-proposal.js`, no ?

Comment (michaelfig) [reply packages/builders/scripts/vats/upgrade-orchestration.js:1]: I'd structure it that way, if you could remind me of the magic of how to supply arguments to an `upgrade.go` proposal step (what I'd write in JSON as `{ "module": "@agoric/builders/scripts/...", "entrypoint": "defaultProposalBuilder", "args": [ &lt;arbitrary JSON> ] }').

Comment (mhofman) [reply packages/builders/scripts/vats/upgrade-orchestration.js:1]: Oh I'm ok with the proposal builder script keeping the hard coding, I was just saying it seems to already be using a proposal that is generic, not specific to orch-core.

TLDR: let's just rename `upgrade-orch-core-proposal.js` ?

Comment (michaelfig) [reply packages/builders/scripts/vats/upgrade-orchestration.js:1]: ~(But not for this PR... just a future generalisation.)~

Comment (michaelfig) [reply packages/builders/scripts/vats/upgrade-orchestration.js:1]: Oh, I see.  Yes, I can do that immediately.

**Review by mhofman — APPROVED** [review]
This looks fine by me. Any reason it's still draft?
